### PR TITLE
[MIG] product_variant_default_code

### PR DESCRIPTION
**Pull request by mikelarre**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0

File: product_variant_default_code/models/product.py [hunk 1]
```diff
@@ -219,9 +219,10 @@ def write(self, vals):
             return super(ProductAttributeValue, self).write(vals)
         result = super(ProductAttributeValue, self).write(vals)
         # Rewrite reference on all product variants affected
-        for product in self.mapped('product_ids').filtered(
-                lambda x: x.product_tmpl_id.reference_mask and not
-                x.manual_code
-                ).mapped('product_tmpl_id.product_variant_ids'):
+        for product in self.env['product.product'].search(
+                [('attribute_value_ids', '=', self.id)]).filtered(
```
Comment (kos94ok-3D) [inline product_variant_default_code/models/product.py]: Can get singletone error.
Use  ('attribute_value_ids', 'in', self.ids)

Comment (kos94ok-3D) [inline product_variant_default_code/models/product.py]: Can get singletone error.
Use ('attribute_value_ids', 'in', self.ids)

**Comment by fuentes73** [comment]
Hi @mikelarre could you check @kos94ok-3D comment?

**Review by fuentes73 — APPROVED** [review]

**Comment by fuentes73** [comment]
@ioans73 @Abranes @Willerr-mo please review

**Review by Abranes — APPROVED** [review]

**Pull request merged**